lib/events.js

```diff
@@ -472,14 +482,14 @@ function onceWrapper() {
     this.target.removeListener(this.type, this.wrapFn);
     this.fired = true;
     if (arguments.length === 0)
-      return this.listener.call(this.target);
-    return this.listener.apply(this.target, arguments);
+      return FunctionPrototypeCall(this.listener, this.target);
+    return FunctionPrototypeApply(this.listener, this.target, arguments);
```

Comment:
`%Reflect.apply%` is native, so possibly faster than binding `%Function.prototype.apply%` to the function.
```suggestion
    return ReflectApply(this.listener, this.target, arguments);
```

Reply:
That's right. Thanks.
I have updated the Commit.